docs/textual_definitions_SOP.md

```diff
@@ -1,70 +1,84 @@
 
 Here are a couple of examples of minimal definitions that are correct but not useful to most users:
 
-1. We could minimally define a corneal endothelial cell as 'Any endothelial cell that is part of the cornea'.  This may well be sufficient for an expert in the anatomy and biology of the cornea, but to most biologists, the term "endothelial cell" brings to mind the principal cell types of lymphatic or blood vessels.  However, "corneal endothelium" refers to a monolayer of flat cells on the underside of the cornea.    
-2. Similarly, a perfectly accurate minimal definition of a type II pneuomocyte is an epithelial cell that has an 'alveolar lamellar body' (a unique structure only found in these cell types). But this is useless information to a user who knows nothing about this structure (many biologists) or who is annotating data that does not resolve this structure.
+We could minimally define a corneal endothelial cell as 'Any endothelial cell that is part of the cornea'. This may well be sufficient for an expert in the anatomy and biology of the cornea, but to most biologists, the term ""endothelial cell"" brings to mind the principal cell types of lymphatic or blood vessels. However, ""corneal endothelium"" refers to a monolayer of flat cells on the underside of the cornea.  
+Similarly, a perfectly accurate minimal definition of a type II pneumocyte is an epithelial cell that has an 'alveolar lamellar body' (a unique structure only found in these cell types). But this is useless information to a user who knows nothing about this structure (many biologists) or who is annotating data that does not resolve this structure.
+
+A second use of ontologies is to encode knowledge in the form of useful formal links between ontology terms. For example, in CL, we record function and cell components via links to gene ontology terms, location via links to CL, and lineage via links to other CL terms. Not all of this information is particularly useful for recognizing a cell type, but it is of use to our users and so we often record it in CL using formal relationships. This is relevant to ontology definitions because it is good practice for formal and textual definitions to match, and textual definitions are the place we encode supporting references. It’s essential that textual definitions capture the assertions made in the formal relationships as closely as possible without becoming stilted and difficult to read.
```

Comment:
```suggestion
A second use of ontologies is to encode knowledge in the form of useful formal links between ontology terms. For example, in CL, we record function and cell components via links to gene ontology terms, location via links to Uberon, and lineage via links to other CL terms. Not all of this information is particularly useful for recognizing a cell type, but it is of use to our users and so we often record it in CL using formal relationships. This is relevant to ontology definitions because it is good practice for formal and textual definitions to match, and textual definitions are the place we encode supporting references. It’s essential that textual definitions capture the assertions made in the formal relationships as closely as possible without becoming stilted and difficult to read.
```